charts/vela-core/templates/admission-webhooks/mutatingWebhookConfiguration.yaml

```diff
@@ -52,6 +52,7 @@ webhooks:
           - core.oam.dev
         apiVersions:
           - v1alpha2
```

Comment:
we can actually move appRollout to v1beta1, no need to support v1alpha2, no one is using it.